dev/reference/sql/statements/split-region.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Split region 使用文档
+category: reference
+---
+
+# Split region 使用文档
+
+在 TiDB 中新建一个表后，默认会单独切分出 1 个 region 来存储这个表的数据，这个默认行为由配置文件中的 `split-table` 控制。当这个 region 中的数据超过默认 region 大小限制后，这个 region 会开始分裂成 2 个 region。
+
+上述情况中，如果在新建的表上发生大批量写入，则会造成热点，因为开始只有一个 region，所有的写请求都发生在该 region 所在的那台 TiKV 上。
+
+为解决上述场景中的热点问题，TiDB 引入了预切分 region 的功能，即可以根据用户指定的参数，预先为某个表切分出多个 region，并打散到各个 TiKV 上去。
+
+## Split region 的使用
+
+Split region 有 2 种不同的语法，具体如下：
+
+{{< copyable "sql" >}}
+
+```sql
+SPLIT TABLE table_name [INDEX index_name] BETWEEN (lower_value) AND (upper_value) REGIONS region_num
+```
+
+`BETWEEN lower_value AND upper_value REGIONS region_num` 语法是通过指定上、下边界和 region 数量，然后在上、下边界之间均匀切分出 `region_num` 个 region。
+
+{{< copyable "sql" >}}
+
+```sql
+SPLIT TABLE table_name [INDEX index_name] BY (value_list) [, (value_list)] ...
+```
+
+`BY value_list…` 语法将手动指定一系列的点，然后根据这些指定的点切分 region，适用于数据不均匀分布的场景。
+
+### Split Table region
+
+表中行数据的 key 由 `table_id` 和 `row_id` 编码组成，格式如下：
+
+```go
+t[table_id]_r[row_id]
+```
+
+例如，当 `table_id` 是 22，`row_id` 是 11 时：
+
+```go
+t22_r11
+```
+
+同一表中行数据的 `table_id` 是一样的，但 `row_id` 肯定不一样，所以可以根据 `row_id` 来切分 region。
+
+#### 均匀切分
+
+由于 `row_id` 是整数，所以根据用户指定的 `lower_value`、`upper_value` 以及 `region_num`，可以很容易地计算出需要切分的 key。先计算出 step（`step = (upper_value - lower)/num`），然后在 `lower` 和 `upper` 之间每隔 step 区间切一次，最终切出 `num` 个 region。
```

Comment:
(upper_value - lower) -> (upper_value - lower_value)
or
(upper_value - lower) -> (upper - lower) ?

Reply:
PTAL. @crazycs520 If there's a mistake, please pull another PR to fix, thanks!

Reply:
fix in https://github.com/pingcap/docs-cn/pull/1509